test/integration/volume/attach_detach_test.go

```diff
@@ -168,6 +170,18 @@ func waitToObservePods(t *testing.T, podInformer cache.SharedIndexInformer, podN
 	}
 }
 
+func waitForPodsInDSWP(t *testing.T, dswp volumecache.DesiredStateOfWorld) {
+	if err := wait.Poll(10*time.Second, 30*time.Second, func() (bool, error) {
```

Comment:
AFAIK, DSW should be pretty quick and we should poll more often to speed up the test. 10 seconds is IMO too long. What about 100ms?